tests/output/ConsoleInteractionTest.py

```diff
@@ -460,8 +461,10 @@ def apply(*args, **kwargs):
     def test_ask_for_actions_and_apply(self):
         failed_actions = set()
         action = TestAction()
+        action2 = DoNothingAction()
```

Comment:
ugh never name variables var, var2, var3. its bad practice.
rather have it: do_nothing_action or fdsaklaction